tests/settings/ConfigurationGatheringTest.py

```diff
@@ -198,12 +198,16 @@ def test_merge_defaults(self):
                 gather_configuration(lambda *args: True,
                                      self.log_printer,
                                      arg_list=['-S', 'value=1', 'test.value=2',
-                                               '-c', escape(temporary, '\\')] +
+                                               '--tags', 'save', '-c',
```

Comment:
move the `'-c', ` back to the next line, so you dont modify that line.

it is also confusing to have the `'-c', ` on this line when its arg is on the next line.